shared/lib_battery_lifetime_calendar_cycle.h

```diff
@@ -70,7 +70,7 @@ struct cycle_state {
     int rainflow_jlt;                       // last index in Peaks, i.e, if Peaks = [0,1], then jlt = 1
     std::vector<double> rainflow_peaks;
 
-    util::matrix_t<double> cycle_counts;   // Cycles, sorted by DOD bins provided in cycling_matrix
+    std::vector<std::vector<double>> cycle_counts;   // Cycles, sorted by DOD bins provided in cycling_matrix
```

Comment:
This comment is no longer strictly true, since the NMC and LMO/LTO variables don't have a cycling matrix. Can you add some text on how the data here varies by lifetime algorithm?

Reply:
Yes, definitely. Updated the comment and clarified also in the SSC table for cmod_battery_stateful.cpp